src/allencell_ml_segmenter/main/i_viewer.py

```diff
@@ -72,3 +72,31 @@ def subscribe_layers_change_event(
         self, function: Callable[[NapariEvent], None]
     ) -> None:
         pass
+
+    @abstractmethod
+    def get_seg_layers(self, layer_list: list[Layer]) -> list[Layer]:
```

Comment:
Thanks for adding comments! Still not sure on one point: Why does this need to take `layer_list` as a param? Shouldn't the viewer already have access to the layer list and apply a filter on that list to get layers that start with `[seg]`?